client/components/jetpack/wpcom-business-at/index.tsx

```diff
@@ -184,7 +184,7 @@ export default function WPCOMBusinessAT( { product }: Props ): ReactElement {
 	useEffect( () => {
 		if ( automatedTransferStatus === COMPLETE && isJetpack ) {
 			dispatch(
-				successNotice( translate( 'Jetpack features are now activated.' ), {
+				successNotice( translate( 'Jetpack Backup is now active' ), {
```

Comment:
Let's not use a hard-coded product string, but use `content.header` instead. Don't forget to add context for the translation as well :)

Reply:
Good point reusing the header for this.